qdrant-landing/content/documentation/concepts/indexing.md

```diff
@@ -49,6 +49,7 @@ Available field types are:
 * `keyword` - for [keyword](../payload/#keyword) payload, affects [Match](../filtering/#match) filtering conditions.
 * `integer` - for [integer](../payload/#integer) payload, affects [Match](../filtering/#match) and [Range](../filtering/#range) filtering conditions.
 * `float` - for [float](../payload/#float) payload, affects [Range](../filtering/#range) filtering conditions.
+* `bool` - for [bool](../payload/#bool) payload, affects [Match](../filtering/#match) filtering conditions (available as of 1.x.x).
```

Comment:
The version is just a placeholder right now, let's remember to update it once we know the release version